example.json

```
@@ -18,6 +18,8 @@
             "_showDebugWindow": false,
             "_suppressErrors": false,
             "_commitOnStatusChange": true,
+            "_comment": "Note that enabling _commitOnAnyChange may have a large impact on the server and should only be used if absolutely necessary. An alternative is to use a lower _timedCommitFrequency value as it will have a lower to impact on the server.",
+            "_commitOnAnyChange": false,
```

```suggestion
            "_commitOnAnyChange": false,
```
this is covered pretty comprehensively in the README and properties.schema, I don't think we need to repeat it a third time.